server/server.go

```diff
@@ -1360,6 +1376,7 @@ func (s *server) LoadSnapshot(_ context.Context, req *rpcpb.LoadSnapshotRequest)
 		logLevel:            s.cfg.LogLevel,
 		reassignPortsIfUsed: req.GetReassignPortsIfUsed(),
 		snapshotsDir:        s.cfg.SnapshotsDir,
+		genesisPath:         req.CustomNetworkGenesisPath,
```

Comment:
just pass nothing of this to load snapshot

Reply:
done